doc/meetings/build-wg-meeting-2017-02-07.md

```diff
@@ -0,0 +1,78 @@
+# Node.js Foundation Build WG Meeting 2017-02-07
+
+* GitHub issue: https://github.com/nodejs/build/issues/607
+* Meeting video: https://www.youtube.com/watch?v=712Y-ytrBgY
+
+* Previous meeting: https://docs.google.com/document/d/1-joUlwaNYirwDlG9sFUyrHKa43ljgRPpj5o-Ygv9JRc
+
+* Next meeting: 14 Feb 2017
+
+
+## Present
+
+* @joaocgreis João Reis 
+* @mhdawson   Michael Dawson
+* @jgergstroem Johan Bergström
+* @piccoloaiutante Michele Capra
+
+## Standup
+
+* João Reis 
+  * Added shared library for engines test to package
+    used in fanned jobs - https://github.com/nodejs/build/issues/596
+  * Working with Kunal Pathak to add Windows Server 2016
+* Michael Dawson
+  * Work on smartos release and test jobs to accommodate new version of V8
+  * Creation of CI jobs for CIGTM, node-report and abi-stable-node
+  * General work to keep AIX clean
+* Johan Bergström
+  * Ansible refactor
+  * Verify backup routines
+  * Jenkins security updates
+  * Alpine/SmartOS inconsistencies
+  * Jenkins host: disk issues
+  * Bottlenecking in general
+  * Add release host for smartos 15
+* Michele Capra
+  * Working on windows ansible script
+  * Updating doc
+  * Carrying work on remina scripts for generating secrets
+
+## Agenda
+
+### file and directory names for downloads:#515
```

Comment:
Yeah, I agree. It should probably also be a list; following other docs ([like this one](https://github.com/nodejs/build/blob/master/doc/meetings/build-wg-meeting-2016-08-30.md)).

Perhaps file a new PR and update all other docs while at it.